Library/Homebrew/deprecate_disable.rb

```diff
@@ -77,10 +77,22 @@ def message(formula_or_cask)
       end
     end
 
-    replacement = if formula_or_cask.deprecated?
-      formula_or_cask.deprecation_replacement
-    elsif formula_or_cask.disabled?
-      formula_or_cask.disable_replacement
+    replacement = if formula_or_cask.disabled?
+      if formula_or_cask.disable_replacement_formula
+        "--formula #{formula_or_cask.disable_replacement_formula}"
```

Comment:
Might be nice to split the assignment of a `formula_or_cask_replacement` and `--formula` or `--cask` into two blocks to DRY this up a bit